src/mem/ruby/common/NetDest.cc

```diff
@@ -38,19 +40,26 @@ namespace ruby
 
 NetDest::NetDest()
 {
-  resize();
+}
+
+NetDest::NetDest(RubySystem *ruby_system)
+{
+    m_ruby_system = ruby_system;
```

Comment:
Should this be like below? Maybe I'm missing something.

```suggestion
NetDest::NetDest(RubySystem *ruby_system) : m_ruby_system(ruby_system)
{
```

Reply:
Made this change